src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/BQSRPipelineSpark.java

```diff
@@ -92,6 +97,9 @@ protected void runTool(final JavaSparkContext ctx) {
                     "as getParallelVariants(List) is broken");
         }
         final JavaRDD<GATKRead> initialReads = getReads();
```

Comment:
Yes, I ran BaseRecalibratorSpark, ApplyBQSRSpark, and ReadsPipelineSpark with 128MB splits and there were no issues.